client/aliases.go

```diff
@@ -28,8 +28,10 @@ import (
 type aliasAction struct {
 	Action string `json:"action"`
 	Snap   string `json:"snap,omitempty"`
-	Target string `json:"target,omitempty"`
+	App    string `json:"app,omitempty"`
 	Alias  string `json:"alias,omitempty"`
+	// AliasOrSnap is used to trigger Do-What-I-Mean unalias action logic
+	AliasOrSnap string `json:"alias-or-snap,omitempty"`
```

Comment:
Feels somewhat confusing. Can we instead enable this mode when both the snap and the are set to the same name, "foo" and "foo"?  This would somewhat clearly reflect the idea of "remove either".